src/item.cpp

```diff
@@ -9131,9 +9131,12 @@ iteminfo::iteminfo( const std::string &Type, const std::string &Name, double Val
 }
 
 iteminfo vol_to_info( const std::string &type, const std::string &left,
-                      const units::volume &vol, int decimal_places )
+                      const units::volume &vol, int decimal_places, bool lower_is_better )
 {
-    iteminfo::flags f = iteminfo::lower_is_better | iteminfo::no_newline;
+    iteminfo::flags f = iteminfo::no_newline;
+    if( lower_is_better ) {
+        f |= iteminfo::lower_is_better;
```

Comment:
mostly did that to avoid having to change the default behavior of the method ... i ll take a look at how often it is used during the weekend ... maybe i can just change them all to pass along a set of flags in the param list instead.